decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -27,6 +29,11 @@ def accepts_new_comments?
       def comments_have_votes?
         true
       end
+
+      # Public: Returns the number of times an specific project have been checked out.
+      def orders_count
```

Comment:
Shouldn't this be `confirmed_orders_count`? (or finished orders).

Also the `where.not(checked_out_at: nil)` maybe should be a scope at `Order`